src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/AppointmentService.kt

```diff
@@ -70,25 +73,43 @@ class AppointmentService(
     )
   }
 
+  fun createAppointment(createAppointmentSpecification: CreateAppointmentSpecification): AppointmentCreatedDto {
+    val event = prisonApiService.postAppointment(
+      createAppointmentSpecification.bookingId!!,
+      CreateBookingAppointment(
+        locationId = createAppointmentSpecification.locationId,
+        startTime = createAppointmentSpecification.startTime.toString(),
```

Comment:
strange conversions to strings?

Reply:
I did want to change the fields on CreateBookingAppointment to LocalDateTime but it it's used in a few places so left it as is.